seacrowd/sea_datasets/muse/muse.py

```diff
@@ -0,0 +1,161 @@
+                description=f"{_DATASETNAME} SEACrowd schema",
+                schema="seacrowd_t2t",
+                subset_id=f"{_DATASETNAME}_tl_en",
+            ),
+        ]
+        + [
+            SEACrowdConfig(
+                name=f"{_DATASETNAME}_{src_lang}_{tgt_lang}_source",
+                version=datasets.Version(_SOURCE_VERSION),
+                description=f"{_DATASETNAME} source schema",
+                schema="source",
+                subset_id=f"{_DATASETNAME}_{src_lang}_{tgt_lang}",
+            )
+            for src_lang in configs
+            for tgt_lang in configs[src_lang]
+        ]
+        + [
+            SEACrowdConfig(
+                name=f"{_DATASETNAME}_{src_lang}_{tgt_lang}_seacrowd_t2t",
+                version=datasets.Version(_SOURCE_VERSION),
+                description=f"{_DATASETNAME} SEACrowd schema",
+                schema="seacrowd_t2t",
+                subset_id=f"{_DATASETNAME}_{src_lang}_{tgt_lang}",
+            )
+            for src_lang in configs
+            for tgt_lang in configs[src_lang]
+        ]
+    )
+
+    DEFAULT_CONFIG_NAME = f"{_DATASETNAME}_source"
+
+    def _info(self) -> datasets.DatasetInfo:
+
+        if self.config.schema == "source":
+            features = datasets.Features({"id": datasets.Value("string"), "src_text": datasets.Value("string"), "tgt_text": datasets.Value("string")})
+
+        elif self.config.schema == "seacrowd_t2t":
+            features = schemas.text2text_features
+
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=features,
+            homepage=_HOMEPAGE,
+            license=_LICENSE,
+            citation=_CITATION,
+        )
+
+    def _split_generators(self, dl_manager: datasets.DownloadManager) -> List[datasets.SplitGenerator]:
+
+        _, src_lang, tgt_lang = self.config.subset_id.split("_")
+        train_url = _TRAIN_URL_TEMPLATE.format(src=src_lang, tgt=tgt_lang)
+        test_url = _TEST_URL_TEMPLATE.format(src=src_lang, tgt=tgt_lang)
+
+        train_file = dl_manager.download_and_extract(train_url)
+        test_file = dl_manager.download_and_extract(test_url)
+
+        return [
+            datasets.SplitGenerator(
+                name=datasets.Split.TRAIN,
+                gen_kwargs={"src_lang": src_lang, "tgt_lang": tgt_lang, "filepath": train_file},
+            ),
+            datasets.SplitGenerator(
+                name=datasets.Split.VALIDATION,
+                gen_kwargs={"src_lang": src_lang, "tgt_lang": tgt_lang, "filepath": test_file},
+            ),
+        ]
+
+    def _generate_examples(self, src_lang: str, tgt_lang: str, filepath: Path) -> Tuple[int, Dict]:
+        if self.config.schema == "source":
+            for row_id, line in enumerate(open(filepath)):
+                src_text, tgt_text = line.strip().split("\t")
+                yield row_id, {"id": row_id, "src_text": src_text, "tgt_text": tgt_text}
```

Comment:
Can we use the 3-letter ISO codes instead of 2-letter codes for the subset names, `text_1_name` values, and `text_2_name` values?